lib/game_teams.rb

```diff
@@ -20,7 +20,7 @@ def self.win_percentage(grouped = @@all_game_teams_data.group_by(&:team_id))
     grouped = grouped
     grouped.map {|team, data| grouped[team] = data.map(&:result)}
     grouped.map do |team, data|
-      grouped[team] = (data.count("WIN")/data.size.to_f).round(2)
+      grouped[team] = (data.count("WIN")/data.size.to_f).round(5)
```

Comment:
The percentage discrepancies between teams was larger than two decimal places so only one team was matching the condition. Rounding to an arbitrary five decimal places takes care of that and makes it more reasonable to test.